packages/client/modules/userDashboard/components/GcalModal/GcalModal.tsx

```diff
@@ -93,6 +117,52 @@ const GcalModal = (props: Props) => {
     onChange(e)
   }
 
+  const onInvitesChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const nextValue = e.target.value
+    if (rawInvitees === nextValue) return
+    const {parsedInvitees, invalidEmailExists} = parseEmailAddressList(nextValue)
+    const allInvitees = parsedInvitees
+      ? (parsedInvitees.map((invitee: any) => invitee.address) as string[])
+      : []
+    const uniqueInvitees = Array.from(new Set(allInvitees))
+    if (invalidEmailExists) {
+      const lastValidEmail = uniqueInvitees[uniqueInvitees.length - 1]
+      lastValidEmail
+        ? setInviteError(`Invalid email(s) after ${lastValidEmail}`)
+        : setInviteError(`Invalid email(s)`)
+    } else {
+      setInviteError(null)
+    }
+    setRawInvitees(nextValue)
+    setInvitees(uniqueInvitees)
+  }
+
+  const handleToggleInviteAll = () => {
+    if (!inviteAll) {
+      const {parsedInvitees} = parseEmailAddressList(rawInvitees)
+      const currentInvitees = parsedInvitees
+        ? (parsedInvitees.map((invitee: any) => invitee.address) as string[])
```

Comment:
Can we specify a type instead of `any`?

Reply:
This was copy & pasted, but I've added a type 👍